test/aqua/operators/test_op_construction.py

```diff
@@ -960,27 +960,28 @@ class TestListOpMethods(QiskitAquaTestCase):
     def test_indexing(self):
         """Test indexing and slicing"""
         coeff = 3 + .2j
-        states_op = ListOp([X, Y, Z, I], coeff=coeff)
+        classes = [ListOp, SummedOp, ComposedOp, TensoredOp]
```

Comment:
Looks great - but could I ask you to take a look at using ddt to have this list of classes done via that instead of a for loop inside the test case. Using a ddt annotation it can take say a list of parameters and then calls the test case over the list (so you don't need to do a for loop in the test case) and each test case will show up with a unique name (as expanded by the annotation at runtime) so that its easier to figure what the failure relates too. In other words the test case would be changed to take a class param that ddt can supply via the annotation. If you look at the test cases in Aqua you will find quite a few with ddt as examples.